### DISCUSSION TITLE
Individual menu customization

### DISCUSSION THREAD

**Discussion by Dano7762** [Open-Shell/Open-Shell-Menu#1737] [Q&A] [answered]
Hello
Is it possible to customize the menu for each user of the same PC?
For example, if I add a folder to the list of all programs, is it possible for this to affect only the user concerned or not?
At the moment it affects all users of the PC.

**Comment by ge0rdi** [comment, marked as answer]
> For example, if I add a folder to the list of all programs, is it possible for this to affect only the user concerned or not?

You can add folder/shortcut to user's Programs folder `%APPDATA%\Microsoft\Windows\Start Menu\Programs`.
To open it right-click on Start button and select `Open`.

**Reply by Dano7762** [reply]
Thank you very much !
